src/cdomains/apron/affineEqualityDomain.apron.ml

```diff
@@ -17,6 +17,16 @@ sig
   val cmp : t -> t -> int
   val to_string:  t -> string
   val of_int: int -> t
+  val get_den: t -> IntOps.BigIntOps.t
+  val get_num: t -> IntOps.BigIntOps.t
+end
+
+module Mpqf = struct
+  include Mpqf
+
+  let get_den x = IntOps.BigIntOps.of_string @@ Mpzf.to_string @@ Mpqf.get_den x
+
+  let get_num x = IntOps.BigIntOps.of_string @@ Mpzf.to_string @@ Mpqf.get_num x
```

Comment:
Just wondering, is your branch so far behind `master` that it doesn't have the `Z_mlgmpidl` module? Because the latter provides a way to convert between these without having to go through strings.